### PR TITLE
rosdep install - support for additional python dependencies (Ubuntu Focal)

### PR DESCRIPTION
**Pull request by up201305618**
While developing with _noetic_ I had difficulties installing some _python_ dependencies with _rosdep_.
Added support for those dependencies.

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -882,6 +882,7 @@ python-catkin-pkg:
   ubuntu:
     artful: [python-catkin-pkg]
     bionic: [python-catkin-pkg]
+    focal: [python3-catkin-pkg]
```
Comment (cottsay) [inline rosdep/python.yaml:885]: This is a Python 3 package, but it is a Python 2 key. From what I can tell, there is no Python 2 package for `catkin_pkg` in Focal.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -1649,6 +1650,7 @@ python-future:
   ubuntu:
     artful: [python-future]
     bionic: [python-future]
+    focal: [python3-future]
```
Comment (cottsay) [inline rosdep/python.yaml:1653]: Again, this is a python 3 package on a Python 2 key, and again, I don't think there exists a Python 2 package to satisfy this dependency.

**Review by cottsay — CHANGES_REQUESTED** [review]
ROS Noetic uses Python 3 packages. If you have encountered a Noetic package using these keys, the package should be updated to use the Python 3 rosdep keys. Python 3 package shouldn't be added to rules for Python 2 keys.

Also, as described in the [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions), please include links to the packaging system for the rules you're proposing.

File: rosdep/python.yaml (hunk 1 above)
Comment (ivanpauno) [reply rosdep/python.yaml:885]: @up201305618 if there's no python 2 version of the package for focal, it cannot be added.
See the [contributing guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md).

**Comment by tfoote** [comment]
I'm going to close this PR because it now conflicts and hasn't had an update in 3 weeks. Please either update the PR to resolve the comments and rebase then comment back to reopen, or open a new PR with the rebased fixes and link it back to here.

**Pull request closed without merging**